app/controllers/admin/undo_decision_controller.rb

```diff
@@ -0,0 +1,42 @@
+class Admin::AmendmentsController < Admin::BaseAdminController
```

Comment:
Did you mean to commit this file?

Reply:
Nope, spotted it just before you commented!

Reply:
Ah 🙌